app/views/entities/new.html.erb

```diff
@@ -1,9 +1,11 @@
-<h1>New entity</h1>
+<h1 class="text-center">New entity</h1>
+<div class="text-center mt-4">
+<%= render "entities/form", {entity: @entity, category: @category} %>
 
-<%= render "form", entity: @entity %>
+</div>
 
 <br>
-
-<div>
-  <%= link_to "Back to entities", entities_path %>
+<div class="text-center ms-4 mt-4">
+  <%= link_to "Back to entities", category_entities_path(@category), class:"btn btn-primary"
+ %>
 </div>
```

Comment:
- [ ] Kindly style the entity for using the design.

  Your Implementation | Behance Design
  :---:|:---:
  ![rroe](https://user-images.githubusercontent.com/54848363/214605886-1478bb63-acc9-44a3-8d3f-937dbb1ccedc.jpg) | ![siow](https://user-images.githubusercontent.com/54848363/214605891-05b2edbc-853b-45eb-ac4b-d31f23ec223e.jpg)